-- [ERROR]  [2015-09-11 09:38:24,655]
-- [CoreAlgorithm.TaskManager.ForecastClass::Void RunSelfLearning()]
-- [文件]:D:\SinteringProControl\Platform\PublicClass\LogClass.cs--第27行
-- [详细]:外部组件发生异常。
********************************************************
